shared/teams/team/header/index.js

```diff
@@ -61,11 +68,19 @@ const _TeamHeader = (props: Props) => (
       {/* Description */}
       {!props.loading && (props.canEditDescription || props.description) ? (
         <Text
-          style={{
-            paddingTop: globalMargins.tiny,
-            color: props.description ? globalColors.black_75 : globalColors.black_20,
-            maxWidth: 560,
-          }}
+          className={props.description ? 'hover-underline' : ''}
+          style={collapseStyles([
+            {
+              color: props.description ? globalColors.black_75 : globalColors.black_20,
+              maxWidth: 560,
+              paddingTop: globalMargins.tiny,
+            },
+            platformStyles({
```

Comment:
we have globalStyles.clickable for this

Reply:
Assuming you mean `desktopStyles.clickable`, that only gives me the `pointer`, but I need the `text` cursor when a description is present.

Reply:
ah, we should add that to desktopStyles then you could do
...(props.description ? desktopStyles.editable : desktopStyles.clickable)

Reply:
Will do!